definitions/procedures/restore/configs.rb

```diff
@@ -19,10 +19,15 @@ def run
       end
     end
 
+    # rubocop:disable  Metrics/MethodLength
     def restore_configs(backup)
       exclude = ForemanMaintain.available_features.each_with_object([]) do |feat, cfgs|
+        if online_backup?
+          feat.config_files_exclude_for_online.each { |f| cfgs << f.gsub(%r{^/}, '') }
```

Comment:
I think, it would be good to have a single method instead of two i.e config_files_to_exclude & config_files_exclude_for_online. 
Refactoring will be required but it won't block this PR.

Reply:
Agree, we shall refactor this.

Reply:
Created re-factoring issue - https://projects.theforeman.org/issues/29880 just for future tracking